### ISSUE TITLE
TF docker 1.4.3 - terraform not found

### ISSUE THREAD

**Issue by a-nomad** [hashicorp/terraform#32947]
### Terraform Version

```shell
1.4.3
```

### Terraform Configuration Files

default

### Debug Output

-

### Expected Behavior

must working

### Actual Behavior

`exec /bin/terraform: no such file or directory`

### Steps to Reproduce

`terraform`
or
`whereis terraform`

### Additional Context

official docker terraform v1.4.3

**Comment by moskeyombus** [comment]
Doesn't seem to be limited to 1.4.3, I reverted to 1.4.2 and experienced the same result

**Comment by tonglil** [comment]
Seems to work on m1, but not on intel.
Same image digest.

**Comment by lemelop** [comment]
+1
I'm facing the same situation with Linux Intel

**Comment by Starttoaster** [comment]
Running this on an intel amd64 linux server:
```
docker run -it --rm --entrypoint="/bin/sh" hashicorp/terraform:latest
/ # which terraform
/bin/terraform
/ # terraform
/bin/sh: terraform: not found
/ #
```
Pretty sure the latest release compiled terraform for the wrong arch in the linux/amd64 version of the image

**Comment by crw** [comment]
Thanks for this report! The release engineering team is aware of this issue and is currently working to resolve it. Thanks for your patience.

**Comment by crw** [comment]
Quick update: the `latest` tag is currently back to referencing the known-good build at `1.4.2` while the team debugs the issue with the `1.4.3` build.

**Comment by Boulet-** [comment]
If it helps:
```
terraform_1.4.2:                 ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=tFekH7Rjd6o8zcpM-k9c/VCHDqfYTow6DxtT1vkKv/LWCu5FIIZbMZovHeFY1R/ccU_xekDPd5S6PvhgaH4, stripped
terraform_1.4.3:                 ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=gkYWTpFtg9abMJLCXsZF/5m7vIEZ-pSIKdRzNZdQp/md5RAdffd2LoZZjUjqLB/ST1JRA1q3reyPpNMvxtg, stripped
```

The binary from `https://releases.hashicorp.com/terraform/1.4.3/terraform_1.4.3_linux_amd64.zip` is not a static binary since 1.4.3.

**Comment by crw** [comment]
Thanks - there is a separate incident for the incorrect linking issue. https://status.hashicorp.com/incidents/dc5gwm249ryc

**Comment by crw** [comment]
The incident is resolved, and `1.4.4` is now available. 

[status.hashicorp.com/incidents/dc5gwm249ryc](https://status.hashicorp.com/incidents/dc5gwm249ryc)

https://releases.hashicorp.com/terraform/1.4.4/

Thanks very much to everyone for the reports in this thread.
